### PR TITLE
Toast functionality

### PR DESCRIPTION
**Pull request by yaswanthmyneni**
# Create a Toast functionality

## Preview: [lucent-video-lib](https://deploy-preview-9--ym-video-lib-dev.netlify.app/)
-----> Click on the categories on the homepage to visit the video listing page in the preview.
-----> On clicking the watch now button, it will be redirected to the video page and also adds the data to the history page at the same time.
-----> login before adding any data to watch later, playlist, history, like pages.

### CODE REVIEW

**Review by omkarpatke — APPROVED** [review]
all things are good, feel free to merge.

**Review by bajajcodes — COMMENTED** [review]
- Good use of comments, as mentioned annotation TODO.
- Simple implementation, which maade code readable and consistent.
- File wishlist-functions.js was missing.

_Good to merge._

File: src/components/toast/Toast.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+const Toast = () => {
+  // from toast context
+  const {
+    toastState: { toastList },
+    toastDispatch,
+  } = useToastContext();
+
+  useEffect(() => {
+    const intervalId = setInterval(() => {
```
Comment (kgajriaAgarwal) [inline src/components/toast/Toast.jsx]: Suggestion: You can try using setTimeout  for toast

**Review by kgajriaAgarwal — APPROVED** [review]
Clean and robust code
can be merged !

**Review by Sahil25061999 — APPROVED** [review]
Looks good to me, nice use of comments in the playlist file

**Pull request merged**